app/components/Carousel.tsx

```diff
@@ -50,7 +53,9 @@ export const SPACER_WIDTH = (width - CAROUSEL_IMAGE_WIDTH) / 2
 const AnimatedFlatList = Animated.createAnimatedComponent(FlatList)
 
 const openLink = (destination: string) =>
-  destination.startsWith("https") ? openLinkInBrowser(destination) : openMap(destination)
+  destination.startsWith("http") || destination.startsWith("https")
```

Comment:
Could you just check for "http" since "https" would match on that too?

Reply:
<img src="https://media.giphy.com/media/egbhUNcdOYI8bXEAi7/giphy-downsized.gif">